File: CS-3320/HW/PC7/factorial.java
public class factorial {

   public static void main (String [] args) {
	int num = 5;
	int res = 1;

	while(num > 0 ){
		res=res*num;
		num--;		
	}
	
	System.out.println("Factorial of 5 is: " + res);
   }
}

File: CS-3320/HW/PC7/factorial.c
#include <stdio.h>
 
int main(void)
{
  
        int num = 5;
	int num2 = num;
        int res = 1;

        while(num2 > 0 ){
                res=res*num2;
                num2--;          
        }

	printf("The factorial of %d is %d\n",num, res);
        return 0;
}

